weldx/utility.py

```diff
@@ -13,6 +13,41 @@
 
 import weldx.transformations as tf
 from weldx.constants import WELDX_QUANTITY as Q_
+from weldx.core import MathematicalExpression, TimeSeries
+
+
+def _sine(
```

Comment:
why "private" (`_`)?

Reply:
I think we should "publish" functions like this later in a more concise way (like a "math" module or similar)

So for now nice to have but not really kind of the public API